### PR TITLE
refactor: use config file for rateLimit

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

File: packages/api-server/rateLimit.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const { AccessGuard } = require("./lib/cache/guard");
```
Comment (classicalliu) [inline packages/api-server/rateLimit.js:1]: Can we move this file to src and write in TypeScript ?

And rename file name to `rate-limit.ts` ?

File: README.md [hunk 2]
```diff
@@ -53,6 +53,20 @@ $ cat > ./packages/api-server/allowed-addresses.json <<EOF
...
 ```
 
+rate limit config
+
+```bash
+$ cat > ./packages/api-server/rate-limit-config.json <<EOF
+{
+  "expired_time": 60000,
+  "methods": {
+    "poly_executeRawL2Transaction": 30,
```
[the remark this answers is not in the document]
Comment (RetricSu) [reply README.md:63]: like below said, max requests number in expired_time

File: packages/api-server/rateLimit.js (hunk 1 above)
Comment (RetricSu) [reply packages/api-server/rateLimit.js]: updated

File: packages/api-server/src/rate-limit.ts [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
-async function rateLimit(req, res, next, rpcMethod, reqId) {
-  if (hasMethod(req.body, rpcMethod) && reqId != null) {
+export async function rateLimit(
+  req: Request,
+  res: Response,
+  next: NextFunction,
+  rpcMethod: string,
+  reqId: string | undefined
+) {
+  if (hasMethod(req.body, rpcMethod) && reqId != undefined) {
```
Comment (classicalliu) [inline packages/api-server/src/rate-limit.ts:26]:
```suggestion
  if (hasMethod(req.body, rpcMethod) && reqId != null) {
```

**Review by classicalliu — APPROVED** [review]

**Pull request merged**